File: articles/the-history-of-programming-languages/go/language_trait_summary.go
package main

import "fmt"

func main() {
	languages := map[string]string{
		"Fortran": "scientific numerical programming",
		"Lisp":   "symbolic computation",
		"SQL":    "declarative data querying",
		"Rust":   "memory-safe systems programming",
	}
	for language, trait := range languages {
		fmt.Printf("%s: %s\n", language, trait)
	}
}

File: articles/the-history-of-programming-languages/julia/language_trait_summary.jl
languages = Dict(
    "Fortran" => "scientific numerical programming",
    "Lisp" => "symbolic computation",
    "SQL" => "declarative data querying",
    "Rust" => "memory-safe systems programming"
)

for (language, trait) in languages
    println(language, ": ", trait)
end
